### PR TITLE
change docker branch

### PR DESCRIPTION
**Pull request by Rzeps0n**
Let's make the docker branch contain only files required for building docker image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
-FROM golang:1.18.4 as base
```
Comment (Pixel48) [inline Dockerfile:1]: This version was provided by upstream's installation manual - @Rzeps0n have you tested if project builds on top of alpine?

**Review by Pixel48 — COMMENTED** [review]
golang:alpine deps should not change over time, but have this version been tested on new base image?

In regard to mass deletions - I understand that's for `--signle-branch` convenience(?) If remerged back into main it could cause some problems

File: Dockerfile (hunk 1 above)
Comment (Rzeps0n) [reply Dockerfile:1]: i did
it works perfectly while having less vulnerabilities according to docker scout

**Comment by Rzeps0n** [comment]
> golang:alpine deps should not change over time, but have this version been tested on new base image?
> 
> In regard to mass deletions - I understand that's for `--signle-branch` convenience(?) If remerged back into main it could cause some problems

I'd like the docker branch to remain seperate from main, yes

**Comment by Pixel48** [comment]
Everything resolved - merging via rebase to keep linear history

**Pull request merged**